lib/presentation/screens/auth/widgets/login.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:truckbill/presentation/router/app_router.dart';
+import 'package:truckbill/presentation/theme/app_typography.dart';
+import 'package:truckbill/utils/context_extensions.dart';
+
+class Login extends StatelessWidget {
+  const Login({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return TextButton(
+      onPressed: () {
+        SignInRoute().go(context);
+      },
```

Comment:
Można też odrobinie uprościć jeśli mamy wywołanie tylko jednej metodki

```suggestion
      onPressed: () => SignInRoute().go(context),
```

Reply:
Tak samo w inncyh miejscach możesz popatrzeć